lib/common/models/redis_store.rb

```diff
@@ -78,7 +78,7 @@ def self.pop(redis_key = nil)
     end
 
     def self.exists?(redis_key = nil)
-      redis_namespace.exists(redis_key)
+      redis_namespace.exists?(redis_key)
```

Comment:
fakeredis introduced the `exists?` method, so this method was now returning `1` instead of true. Adding the `?` gave us passing tests.

Will need to smoke test in staging to confirm it's working as expected